membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -63,6 +65,7 @@ class TouchpointComponents(stage: String)(implicit system: ActorSystem) {
   lazy val contactRepo = new SimpleContactRepository(tpConfig.salesforce, system.scheduler, Config.applicationName)
   lazy val attrService: AttributeService = new ScanamoAttributeService(new AmazonDynamoDBAsyncClient(com.gu.aws.CredentialsProvider).withRegion(Regions.EU_WEST_1), dynamoAttributesTable)
   lazy val behaviourService: BehaviourService = new ScanamoBehaviourService(new AmazonDynamoDBAsyncClient(com.gu.aws.CredentialsProvider).withRegion(Regions.EU_WEST_1), dynamoBehaviourTable)
+  lazy val featureToggleService: ScanamoFeatureToggleService = new ScanamoFeatureToggleService(new AmazonDynamoDBAsyncClient(com.gu.aws.CredentialsProvider).withRegion(Regions.EU_WEST_1), dynamoFeatureToggleTable)
```

Comment:
really, the TouchpointComponents class is some kind of epic class of all knowledge about the world.  unfortunately this kind of omniscient class is a bit of an antipattern in java, and more so in scala, so ideally I'd like to avoid adding to it.  Basically everything depends on it, and it depends on everything.

In this specific case, the only thing that might have state (some kind of cached connection?) is the AmazonDynamoDBAsyncClient, so that's the only thing we need to maintain a reference to.  The rest can all be objects containing functions but no state, which are referenced directly when they are needed in the code.